spec-0013/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "SPEC 13 — Recommended targets and naming conventions"
+number: 13
+date: 2024-06-05
+author:
+  - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matthias Bussonnier <bussonniermatthias@gmail.com>"
+  - "Jarrod Millman <millman@berkeley.edu>"
+discussion: https://discuss.scientific-python.org/t/spec-13-recommended-targets-and-naming-conventions
+endorsed-by:
+---
+
+## Description
+
+For consistency and decreased cognitive load across the ecosystem, this SPEC
+recommends naming conventions around various project aspects--such as project
+structure, repository layout, folder names, task runner and `pyproject.toml`
+targets name.
+
+From a cursory survey in the Scientific Python ecosystem, we discover some
+frustration from contributors and maintainer when moving from one project to
+another and believe that consistency will make it both easier for existing
+maintainer to contribute to project as well a decrease the confusion of new
+developers when contributing or creating new projects.
+
+There seem to be a strong consensus with preference for `docs` in favor of
+`docs`, and a preference for `tests` in favor of `test`.
```

Comment:
```suggestion
There seem to be a strong consensus with preference for `docs` over
`doc`, and a preference for `tests` over `test`.
```
"preference for tests" means that "tests" is preferred, but "in favor of test" means that "test" is preferred, so it was not clear to me which was preferred.